src/components/TextField/README.md

```diff
@@ -774,3 +774,83 @@ class TextFieldExample extends React.Component {
+
+- [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessibility/)
+- [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+
+<!-- /content-for -->
+
+<!-- content-for: web -->
+
+### Structure
+
+Screen readers convey information about text fields automatically through native HTML.
+
+- Use the `disabled` prop to add the HTML `disabled` attribute to the text field
+- Use the `readOnly` prop to add the HTML `readonly` attribute to the text field
+- If you use the `type` prop, then some technologies update the software keyboard provided to the user to be more specific to the task at hand. This helps merchants with mobility, vision, and cognitive issues to enter information more easily
+
+You can use the `id` prop to provide a unique `id` attribute value for the text field. The component will generate one automatically if you don’t provide one. All text fields must have unique `id` values.
+
+### Labeling
+
+The `label` prop is required to convey the purpose of the checkbox to all merchants.
+
+If there are separate visual cues that convey the purpose of the text field to sighted merchants, then the label can be visually hidden with the `labelHidden` prop.
+
+When you provide help text via the `helpText` prop or an inline error message via the `error` prop, the help or error content is conveyed to screen reader users with the `aria-describedby` attribute. This attribute causes the content to be read along with the label, either immediately or after a short delay.
+
+Use the `placeholder` prop to provide additional instructions. However, don’t rely on placeholders alone since the content isn’t always conveyed to all merchants.
+
+<!-- usageblock -->
+
+#### Do
+
+- Use the label to provide instructions critical to using the text field
+- Use help text and placeholder text to provide additional, non-critical instructions
+
+#### Don’t
+
+Use the placeholder to provide information that’s required to use the text field.
+
+<!-- end -->
+
+### Keyboard support
+
+Text fields have standard keyboard support.
+
+- Merchants who rely on the keyboard expect to be able to move focus to each text field using the <kbd>tab</kbd> key (or <kbd>shift</kbd> + <kbd>tab</kbd> when tabbing backwards)
+- If the `type` is set to `number`, then merchants can use the up and down arrow keys to adjust the value typed into the field
+- Using the `disabled` prop will prevent the text field from receive keyboard focus or inputs
+- Using the `readOnly` prop will allow focus on the text field but prevent input or editing
+
+#### Automatically focusing
+
+In general, avoid focusing on fields automatically. However, you can use the `autoFocus` prop to automatically move focus to the text field. This prop is set to `false` by default and should only be used in cases where it won’t force focus to skip other controls or content of equal or great importance importance.
```

Comment:
I think the paragraph flows a little better if we rephrase so that the first sentence is at the end: 

```
Use the `autoFocus` prop to automatically move focus to the text field. This prop is set to `false` by default and should only be used in cases where it won’t force focus to skip other controls or content of equal or greater importance. In general, avoid focusing on fields automatically. 
``` 

That said, if it's important to make sure that folks don't miss the message about it being best to not automatically focus, then maybe we can say something like: 

``` 
Although you can use the `autoFocus` prop to automatically move focus to the text field, it’s generally best to avoid focusing on fields automatically. The `autoFocus` prop is set to `false` by default and should only be used in cases where it won’t force focus to skip other controls or content of equal or greater importance. 
```